marketing-activity/shopify.extension.toml.liquid

```diff
@@ -0,0 +1,27 @@
+[[extensions]]
+type = "marketing_activity"
+name = "{{ name }}"
+handle = "{{ handle }}"
+{% if uid %}uid = "{{ uid }}"{% endif %}
+title = "Your title"
+description = "Your description"
+app_api_url = "https://url.com/api/execute"
+tactic = "ad"
+channel = "social"
```

Comment:
I think we decided to call this `marketing_channel` instead, right?

Reply:
also wondering if we should leave this and the tactic blank, otherwise I feel like people might forget to update